tests/Doctrine/Tests/DBAL/Functional/Schema/OracleSchemaManagerTest.php

```diff
@@ -273,4 +273,81 @@ public function testListTableDateTypeColumns()
         $this->assertSame('datetime', $columns['col_datetime']->getType()->getName());
         $this->assertSame('datetimetz', $columns['col_datetimetz']->getType()->getName());
     }
+
+    public function testCreateSchemaOnLargeNumberOfTables()
```

Comment:
Uhmm this is tricky - both the 'setup' part and the 'assertion' part are kind of db specific

1) the 'setup' is with db specific queries, to simulate tables and assets generated out of DBAL
2) the 'assertion' part is also specific because it ened to be tailored around what is actually introspected - for instance, lowercase column names in Oracle are converted to uppercase, and these are returned as column names by `$schema->getColumn(...)->getName`. MySql will be different. This may be a bug if we want to ensure portability, but not in scope of addressing here IMHO.